data/brands/amenity/fast_food.json

```diff
@@ -14555,7 +14555,7 @@
         "brand:en": "FJ Veggie",
         "brand:wikidata": "Q130458837",
         "brand:zh": "芳珍蔬食",
-        "cuisine": "vegetarian",
```

Comment:
Is there any value that could be added to `cuisine` here, rather than leave it blank?